articles/ml_system_design/real_time_news_search_using_upstash_kafka_and_vector/upstash_ingest/upstash_ingest/test.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+from embeddings import TextEmbedder
+from settings import settings
+from models import CommonDocument, RefinedDocument, ChunkedDocument, EmbeddedDocument
+from .tools import NewsFetcher
+from .logger import get_logger
+import pytest
+
+logger = get_logger(__name__)
+
+
+def _test_news_fetch():
+    fetcher = NewsFetcher()
+    news_api_articles = fetcher.fetch_from_newsapi()
```

Comment:
Shouldn't we move these functions to the test directory? if not, let's name them differently.

Reply:
Dummy file - to be deleted.
Valid tests are in /tests/*.py folder